graylog2-web-interface/src/components/common/EntityListItem.jsx

```diff
@@ -25,12 +25,14 @@ const EntityListItem = React.createClass({
         <Row className="row-sm">
           <Col md={6}>
             <h2>{this.props.title} {titleSuffix}</h2>
-            <div className="item-description">
+            { (this.props.createdFromContentPack || this.props.description) &&
```

Comment:
As a really tiny nit-pick, the space after the curly brace here breaks the [`react/jsx-curly-spacing`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-spacing.md) style rule.